bin/grouper-api

```diff
@@ -29,6 +30,34 @@ class Application(tornado.web.Application):
         super(Application, self).__init__(*args, **kwargs)
 
 
+class SendEmailThread(Thread):
+    def run(self):
+        while True:
+            logging.debug("Sending emails...")
+            try:
+                now = datetime.utcnow()
+                session = Session()
+                emails = session.query(AsyncNotification).filter(
+                    AsyncNotification.sent == False).all()
```

Comment:
To avoid a race, chain with (send_after < now) and preemptively update to sent=True, reverting to sent=False for those emails that hit the exception handler.